src/browserlib/extract-events.mjs

```diff
@@ -160,24 +160,46 @@ export default function (spec) {
   [...document.querySelectorAll("a")]
     .filter(a => !a.closest(informativeSelector) && isFiringLink(a))
     .forEach(a => {
-      const container = a.parentNode;
-      // There can be multiple "fire an event" links in a container
-      // limiting our text parsing to content in between two such links
-      // (or the end of the container if there is only one such link)
+      // Clone and drop possible annotations to avoid extracting asides.
+      // (note the need to temporarily add the cloned node to the document
+      // so that ranges can be used)
+      let apos = 0;
+      for (let i = 0; i < a.parentNode.children.length; i++) {
```

Comment:
this could be
```js
const apos = [...a.parentNode.children].findIndex(a);
```

Reply:
Indeed, will make the change. FWIW, rather `const apos = [...a.parentNode.children].findIndex(c => c === a);`